src/Analyzers/Core/Analyzers/RemoveUnusedMembers/AbstractRemoveUnusedMembersDiagnosticAnalyzer.cs

```diff
@@ -539,6 +549,20 @@ private ArrayBuilder<string> GetDebuggerDisplayAttributeArguments(INamedTypeSymb
                 return builder;
             }
 
+            private ArrayBuilder<string> GetDynamicDependencyAttributeArguments(INamedTypeSymbol namedTypeSymbol)
```

Comment:
how do you know this type isn't null?

Reply:
@CyrusNajmabadi The value passed comes from symbolEndContext.Symbol, which should be non-null.